app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -289,6 +290,21 @@ private void viewDay(boolean reload) {
         updateNavigationMenuSelection();
     }
 
+    private Map<Integer, List<Lecture>> groupLecturesPerRoom(List<Lecture> lectures) {
```

Comment:
:mag: Please add a unit test for the transformation. You might want to use the `@VisibleForTesting` annotation.